index.html

```diff
@@ -2,10 +2,9 @@
   <head>
     <title>Sea of Awesomeness</title>
   </head>
-    <head>
-        <title>My awesome web page</title>
-    </head>
-
-    Hello there, awesome person!
+    <body>
+        <h1>My awesome web page</h1>
+    <img src+"https://avatars0.githubusercontent.com/u/19331862?v=4
```

Comment:
@stacymbush you should have this in your code for the image:

<img src="https://avatars0.githubusercontent.com/u/19331862?v=4">

You have:

<img src+"https://avatars0.githubusercontent.com/u/19331862?v=4

Reply:
Well shoot.

Reply:
So do I just go back to the Code tab and update it?

Reply:
And what does "Resolve conversation" do

Reply:
Yes.  You can go back to the code tab and edit the file.

Reply:
You can resolve the conversation too.  It will take our convo and archive it.